### PR TITLE
:wrench: Support Electron v4

### PR DESCRIPTION
**Pull request by nornagon**
Electron 4 changed the header directory to use the 'headerOnly' format rather than the 'full' node checkout format.

### CODE REVIEW

File: lib/es6/runtimePaths.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ let runtimePaths = {
                 name: "node.lib"
             }],
             tarPath: "node" + "-v" + targetOptions.runtimeVersion + ".tar.gz",
-            headerOnly: false
+            headerOnly: semver.gte(targetOptions.runtimeVersion, '>=4.0.0-alpha')
```
Comment (unbornchikken) [inline lib/es6/runtimePaths.js]: That should be semver.satfisfies, because you've included the gte operator in the semver string.

Comment (nornagon) [reply lib/es6/runtimePaths.js]: Switched to not include the `>=` operator in the semver string

**Comment by rogeriorc** [comment]
@unbornchikken When it will be released?

**Comment by nornagon** [comment]
It'd be great to get an Electron 4-compatible cmake-js!

**Comment by unbornchikken** [comment]
It has been release just now as of 5.0.0.

**Comment by nornagon** [comment]
Fantastic, thanks @unbornchikken!

**Pull request merged**